package-lock.json

```diff
@@ -4448,7 +4448,6 @@
       "version": "1.1.3",
       "resolved": "https://registry.npmjs.org/define-properties/-/define-properties-1.1.3.tgz",
       "integrity": "sha512-3MqfYKj2lLzdMSf8ZIZE/V+Zuy+BgD6f164e8K2w7dgnpKArBDerGYpM46IYYcjnkdPNMjPk9A6VFB8+3SKlXQ==",
-      "dev": true,
```

Comment:
Hmmm, I did not intentionally remove all these `"dev": true` ... is that gonna be an issue?

Reply:
Hmmmmm that is a little concerning, do you just want to checkout the previous version of package-lock and commit that back?

Reply:
oh, I see you did have intentional changes. Not sure what the implications of that "dev" are or why it would change from installing a package.

Reply:
So this is the documentation on the `dev` property: ```If true then this dependency is either a development dependency ONLY of the top level module or a transitive dependency of one. This is false for dependencies that are both a development dependency of the top level and a transitive dependency of a non-development dependency of the top level.
So the ones that got changed are mostly transitive dependencies...they are dependencies of the dependency I added (listed in the `require` section) so they got changed, and then if those have their own dependencies (e.g. `es-abstract` has like 11 dependencies), then those dependencies also got changed